src/Components/edit/Edit.tsx

```diff
@@ -0,0 +1,101 @@
+import axios from "axios";
```

Comment:
I wonder why you need separate add and edit components. The code is 90% the same.